modules/spacelift/worker-pool/variables.tf

```diff
@@ -215,6 +215,8 @@ variable "instance_refresh" {
     preferences = object({
       instance_warmup        = number
       min_healthy_percentage = number
+      skip_matching          = bool
+      auto_rollback          = bool
```

Comment:
Can we set these all as optional?

```
      instance_warmup        = optional(number, null)
      min_healthy_percentage = optional(number, null)
      skip_matching          = optional(bool, null)
      auto_rollback          = optional(bool, null)
```

Reply:
https://github.com/cloudposse/terraform-aws-ec2-autoscale-group#input_instance_refresh

Reply:
For sure! Just pushed those changes. Great idea with that as well